web-api/terraform/modules/batch/docker-image/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "docker-image",
+  "version": "1.0.0",
+  "main": "testScript.js",
+  "scripts": {
+		"deploy": "./deploy-to-ecr.sh us-east-1 && ./deploy-to-ecr.sh us-west-1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "devDependencies": {
+    "@types/archiver": "^6.0.2",
+    "@types/node": "22.5.5"
+  },
+  "dependencies": {
+    "@aws-sdk/client-apigatewaymanagementapi": "3.650.0",
+    "@aws-sdk/client-s3": "3.654.0",
+    "@aws-sdk/client-sqs": "3.654.0",
+    "@aws-sdk/node-http-handler": "3.374.0",
+    "@aws-sdk/s3-request-presigner": "^3.657.0",
+    "archiver": "^7.0.1"
```

Comment:
Do we want to keep these fixed?

Also, general question: should we now update the dependencies readme to include this file as well?